src/aggregate/aggregate_exec.c

```diff
@@ -890,10 +894,9 @@ static void runCursor(RedisModule_Reply *reply, Cursor *cursor, size_t num) {
   }
 
   // update timeout for current cursor read
-  if (req->qiter.rootProc->type != RP_NETWORK) {
-    updateTimeout(&req->timeoutTime, req->reqConfig.queryTimeoutMS);
-    SearchCtx_UpdateTimeout(req->sctx, req->timeoutTime);
-  }
+  updateTimeout(&req->timeoutTime, req->reqConfig.queryTimeoutMS);
```

Comment:
Review explanation: Should not avoid updating the timeout - it is needed for timeout check in COORD later on